packages/suite-desktop/src-electron/processes/BaseProcess.ts

```diff
@@ -67,15 +67,21 @@ class BaseProcess {
      * Stops the process
      */
     async stop() {
-        if (!this.process) {
-            return;
-        }
-
-        if (!this.process.kill()) {
-            this.process.kill('SIGINT');
-        }
-
-        this.process = null;
+        return new Promise(resolve => {
+            if (!this.process) {
+                resolve();
+            }
+
+            this.process.kill();
+
+            setTimeout(() => {
+                if (!this.process.killed) {
+                    this.process.kill('SIGINT');
+                }
+                this.process = null;
+                resolve();
+            }, 3000);
```

Comment:
```suggestion
            }, 10000);
```

Reply:
I'll rewrite this bit. If the process gets killed without forcing it, this will delay the resolution by X seconds for nothing.